types/lhr/settings.d.ts

```diff
@@ -46,6 +46,9 @@ export type ScreenEmulationSettings = {
   disabled: boolean;
 };
 
+import type {Protocol as Crdp} from 'devtools-protocol/types/protocol.js';
+type StorageType = Crdp.Storage.StorageType;
```

Comment:
nit: move import to the top of the file and inline `Crdp.Storage.StorageType` on L82?

I only wrote it this way to fit in a Github Comment :)

Reply:
Thank you